lib/authorizer/basic.js

```diff
@@ -39,14 +39,12 @@ module.exports = {
     /**
      * Signs a request.
      *
-     * @param {RequestAuth} auth
+     * @param {AuthInterface} auth
      * @param {Request} request
      * @param done
      */
     sign: function (auth, request, done) {
-        var params = auth.parameters().toObject();
-
-        if (!params) { return done(); } // Nothing to do if no parameters are present.
+        var params = auth.get(['username', 'password']);
```

Comment:
Why not `params.get('username')` and `params.get('password')` here? It is just two variables right?

Reply:
Agree